cosmwasm/utils.js

```diff
@@ -35,20 +35,21 @@ const uploadContract = async (client, wallet, config, options) => {
             return client.upload(account.address, wasm, uploadFee).then(({ checksum, codeId }) => ({ checksum, codeId, account }));
         })
         .then(({ account, checksum, codeId }) => {
+            const usedSalt = salt || contractName.concat(chainNames);
```

Comment:
```suggestion
            let salt = options.salt || contractName.concat(chainNames);
```
to reuse `salt` name